src/chrome/content/rules/europe1.fr.xml

```diff
@@ -0,0 +1,66 @@
+<!--
+	Redirects to http:
+		europe1.fr
+		www.europe1.fr
+		recherche.europe1.fr
+
+	No known URL working:
+		api.europe1.fr (503)
+		www.api.europe1.fr (503)
+		bonsplans.europe1.fr
+		cdn-podcast.europe1.fr
+		ezteam.europe1.fr
+		pp.ezteam.europe1.fr
+		fichier.europe1.fr
+		dyn.lelab.europe1.fr
+		media.europe1.fr
+		www.migration.europe1.fr
+		www.new.europe1.fr
+		photo.europe1.fr
+		vins.europe1.fr
+		pp.www.europe1.fr (403)
+
+	Bad certificate:
+		archives.europe1.fr
+		christophe-lamarre.blog.europe1.fr
+		des-clics-et-des-claques.blog.europe1.fr
+		helene-favier.blog.europe1.fr
+		lapolitiqueenshort.blog.europe1.fr
+		leshistoiresda.blog.europe1.fr
+		club-news.europe1.fr
+		planiweb.europe1.fr (self-signed)
+		rencontres.europe1.fr
+		
+	Secure connection failed:
+		avantages.europe1.fr
+		bienvenueauclub.europe1.fr
+		elections.europe1.fr
+		generationseurope1.europe1.fr
+		
+	Bad redirection:
+		club.europe1.fr
+		festival-de-cannes.europe1.fr
+		ketb.europe1.fr
+		live.europe1.fr
+		m.europe1.fr
+		meteo.europe1.fr
+		moi-maire.europe1.fr
+		radio.europe1.fr
+		sport.europe1.fr
+		webradio.europe1.fr
+-->
+<ruleset name="Europe1.fr">
```

Comment:
Important parts of the site are not secured. Could you change the name to `Europe1.fr (partial)`?